src/uct/ib/configure.m4

```diff
@@ -378,12 +378,13 @@ AS_IF([test "x$with_ib" = "xyes"],
            ])
        AS_IF([test "x$with_ib_hw_tm" = xexp],
            [AC_DEFINE([IBV_HW_TM], 1, [IB Tag Matching support])
-            AC_CHECK_MEMBERS([struct ibv_exp_create_srq_attr.dc_offload_params],
-                             [AC_DEFINE([IBV_EXP_HW_TM_DC], 1, [DC Tag Matching support])],
-                             [], [#include <infiniband/verbs_exp.h>])
+            AC_CHECK_MEMBERS([struct ibv_exp_create_srq_attr.dc_offload_params], [
+            AC_DEFINE([IBV_HW_TM_DC], 1, [IB DC Tag Matching support])
```

Comment:
remove this macro; check HW_TM in DC code just like in RC

Reply:
We need this define - MOFED 4.1 supports HW TM but not HW TM DC.

Reply:
let's require that for MOFED TM support, we also need DC( e.g MOFED 4.2+)